src/snowflake/snowpark/dataframe_reader.py

```diff
@@ -248,11 +268,13 @@ def csv(self, path: str) -> DataFrame:
         """Specify the path of the CSV file(s) to load.
 
         Args:
-            path: The stage location of a CSV file, or a stage location that has CSV files.
+            path: A path pointing to a local CSV file, a local directory that has CSV files, a CSV file on a stage, or
+            a stage location that has CSV files.
```

Comment:
I remember we need two indents here if an arg has multiple lines. But if the generated doc is ok then it's ok.